node/src/daemon/mod.rs

```diff
@@ -454,6 +454,20 @@ mod tests {
     use std::iter::FromIterator;
     use std::sync::{Arc, Mutex};
 
+    #[test]
+    fn constants_have_correct_values() {
+        let censorables_expected: HashMap<String, usize> = {
+            vec![
+                ("db-password".to_string(), 16),
+                ("consuming-private-key".to_string(), 64),
+            ]
+            .into_iter()
+            .collect()
+        };
+
+        assert_eq!(*CENSORABLES, censorables_expected);
+    }
```

Comment:
Have this in place.

Reply:
Sorry, I don't understand what change I'm asked for.